README.md

```diff
@@ -1,5 +1,151 @@
-# SuluThemeBundle
+<h1 align="center">SuluThemeBundle</h1>
 
-[![Build Status](https://travis-ci.org/sulu/SuluThemeBundle.svg?branch=develop)](https://travis-ci.org/sulu/SuluThemeBundle)
+<p align="center">
+    <a href="https://sulu.io/" target="_blank">
+        <img width="30%" src="https://sulu.io/uploads/media/800x/00/230-Official%20Bundle%20Seal.svg?v=2-6&inline=1" alt="Official Sulu Bundle Badge">
+    </a>
+</p>
 
-This package enables a basic theming for sulu. It uses liip/theme-bundle to extend the discovery feature of twig.
+<p align="center">
+    <a href="https://github.com/sulu/SuluThemeBundle/releases" target="_blank">
+        <img src="https://img.shields.io/github/tag/sulu/SuluThemeBundle.svg" alt="GitHub tag (latest SemVer)">
+    </a>
+    <a href="https://github.com/sulu/SuluThemeBundle/blob/master/LICENSE" target="_blank">
+        <img src="https://img.shields.io/github/license/sulu/SuluThemeBundle.svg" alt="GitHub license">
+    </a>
+    <a href="https://github.com/sulu/sulu/releases" target="_blank">
+        <img src="https://img.shields.io/badge/sulu%20compatibility-%3E=2.0-52b6ca.svg" alt="Sulu compatibility">
+    </a>
+</p>
+<br/>
+
+The **SuluThemeBundle** provides the functionality to add multiple themes for different look and feel using multiple 
+webspaces in the [Sulu](https://sulu.io/) content management system. 
+
+To achieve this, the bundle uses the [SyliusThemeBundle](https://github.com/Sylius/SyliusThemeBundle) to render different
+twig templates and asset files. Each webspace can have it's own theme.
+
+## 🚀&nbsp; Installation and Usage
+
+### Install the bundle
+
+Execute the following [composer](https://getcomposer.org/) command to add the bundle to the dependencies of your 
+project:
+
+```bash
+composer require sulu/theme-bundle
+```
+
+### Enable the bundle 
+
+Enable the bundle by adding it to the list of registered bundles in the `config/bundles.php` file of your project:
+
+```php
+return [
+    /* ... */
+    Sulu\Bundle\ThemeBundle\SuluThemeBundle::class => ['all' => true],
+];
+```
+
+
+### Configure the SyliusThemeBundle
+
+In order to use the bundle you have to add at least the following default configuration:
+
+```yaml
+# ./config/packages/sylius_theme.yaml
+
+sylius_theme:
+    sources:
+        filesystem: ~
+```
+
+By default, the bundle seeks for the themes in the `%kernel.project_dir%/themes` directory and looks for a configuration
+file named `composer.json`. This can be changed via the yaml configuration:
```

Comment:
```suggestion
By default, the bundle seeks for the themes in the `%kernel.project_dir%/themes` directory and looks for a
theme configuration file named `composer.json`. This can be changed via the yaml configuration:
```